README.md

```diff
@@ -83,6 +83,48 @@ Then try cloning the repository:
 $ git clone ssh://git@myhost.com/my/repo /tmp/my-repo
 ```
 
+## Testing
```

Comment:
Consumers of this repository (people trying to deploy on Docker Compose, or pure docker) won't care about this so it shouldn't be in the README but rather in RELEASE.md. I will move.

Reply:
I think the best thing to do would be:

1. Move these instructions to a new `TESTING.md` file.
2. Update these instructions to refer to invoking `.buildkite/test-pure-docker.sh` instead of invoking `vagrant` directly (then you do not need to update `RELEASING.md` with what is effectively the same set of commands to run vagrant up and down.)

Reply:
Full instructions on setting up a local test are now in `TESTING.md`